extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -144,6 +150,13 @@ const buildExpandedViewOverlay = element => htmlFor(element)`
  */
 const MIN_VERTICAL_SPACE = 48;
 
+/**
+ * Limits the amount of vertical space a component can take in a page.
+ * @const {number}
+ * @private
+ */
+const VERTICAL_MAX_HEIGHT_PERCENTAGE = 0.8;
```

Comment:
Where did this requirement come from?  Did we verify the behavior and numbers with @hongwei1990?